sections/sentinels.tex

```diff
@@ -3,6 +3,10 @@
 
 \begin{multicols*}{2}
 
+\textbf{Author:} silence70011
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1233112440322002964/1233112440322002964}{Archon Studio Discord}
```

Comment:
I don't think source is necessary, but in any case it belongs at the bottom.

Reply:
I wanted to fully credit the guy. But we'll see, I contacted him to get his approval and thoughts on how he wants to received credit

Reply:
I wouldn't wait with merging. If the guy decides he doesn't want to be featured here, we'll delete it.